src/Music/Score/Meta/Time.hs

```diff
@@ -203,8 +203,8 @@ isPowerOfTwo n = (n .&. (n -1)) == 0
 {-# INLINE isPowerOfTwo #-}
 
 -- TODO consolidate
-optionLast :: a -> Option (Last a)
-optionLast = Option . Just . Last
+optionLast :: a -> Maybe (Last a)
```

Comment:
Hi @cianoc 

Thank you very much! I have approved and feel free to merge. 

Quick question: #59 also mentions tempo, but this only seems to fix time signatures and fermata, is that correct?